File: C++/CataManzanas.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){

    int tMan;
    cin >> tMan;

    int cont = 0;

    vector<int> CataMan(5);

    for (int i = 0; i < 5; i++)
    {
        cin >> CataMan[i];
        if (tMan == CataMan[i])
        {
            cont++;
        }
        
    }

    cout << cont;


    return 0;   
}

File: Python/CataManzanas.py
def main():
    tMan = int(input())  # Leemos el número objetivo
    CataMan = list(map(int, input().split()))  # Leemos los 5 números como una lista

    cont = 0  # Contador para coincidencias

    for num in CataMan:
        if tMan == num:
            cont += 1

    print(cont)

if __name__ == "__main__":
    main()
